prog/dftb+/lib_dftb/coulomb.F90

```diff
@@ -363,6 +493,51 @@ subroutine addInvRPrime_cluster(deriv, nAtom, coord, deltaQAtom)
 
   end subroutine addInvRPrime_cluster
 
+#:if WITH_SCALAPACK
+
+  !> Calculates the -1/R**2 deriv contribution for all atoms for the non-periodic case, without
+  !> storing anything.
+  subroutine getDInvRClusterBlacs(grid, descAtomSqr, localShape, coord, deltaQAtom, deriv)
+
+    !> BLACS grid of the distributed derivative vector
+    type(blacsgrid), intent(in) :: grid
+
+    !> Descriptor for an nAtom x nAtom matrix distributed on the grid
+    integer, intent(in) :: descAtomSqr(DLEN_)
+
+    !> Local shape of the distributed nAtom x nAtom matrix
```

Comment:
Local shape of the distributed nAtom x nAtom electrostatic matrix

Reply:
Done.